packages/mgt-components/src/components/mgt-todo/mgt-todo.ts

```diff
@@ -280,95 +335,50 @@ export class MgtTodo extends MgtTasksBase {
    * @memberof MgtTodo
    */
   protected renderTask(task: TodoTask) {
-    const context = { task, list: this._currentList };
+    const context = { task, list: this.currentList };
 
     if (this.hasTemplate('task')) {
       return this.renderTemplate('task', context, task.id);
     }
 
     const isCompleted = (TaskStatus as any)[task.status] === TaskStatus.completed;
-    const isLoading = this._loadingTasks.includes(task.id);
-    const taskCheckClasses = {
-      Complete: !isLoading && isCompleted,
-      Loading: isLoading,
-      TaskCheck: true,
-      TaskIcon: true
-    };
 
-    const taskCheckContent = isLoading
-      ? html`
-          
-        `
-      : isCompleted
+    let taskDetailsTemplate = null;
+
+    const taskDueTemplate = task.dueDateTime
       ? html`
-          
+          <fluent-button appearance="stealth" class="TaskDue">
```

Comment:
What does this button do? 
If there's no click behavior and this is just for styling then we should be looking to implement with another html element and style this appropriately